sdk/test/UnitTests/AWSXRayRecorder.UnitTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net452</TargetFrameworks>
```

Comment:
No, just forgot to change it back after testing, any specific reason that we should keep the framework order in unit test library?

Reply:
I think having different order in unit test project from the dependency project like core, and handlers causes issues when building the project in VisualStudio since VS loads the projects based on the first target framework specified.

Reply:
It's already changed back